pkg/initializer/validator/types.go

```diff
@@ -34,14 +34,18 @@ const (
 	DataDirectoryCorrupt
 	// DataDirectoryStatusUnknown indicates validator failed to check the data directory status.
 	DataDirectoryStatusUnknown
-	// RevisionConsistencyError indicates current etcd revision is inconsistent with latest snapshot revision.
+	// EtcdRevisionConsistencyError indicates current etcd revision is inconsistent with latest snapshot revision.
+	EtcdRevisionConsistencyError
+	// RevisionConsistencyError indicates current etcd revision is inconsistent with latest snapshot revision even after WALs file flushed the data to Bolt DB.
 	RevisionConsistencyError
 	//FailBelowRevisionConsistencyError indicates the current etcd revision is inconsistent with failBelowRevision.
 	FailBelowRevisionConsistencyError
 )
 
 const (
-	snapSuffix = ".snap"
+	snapSuffix            = ".snap"
+	connectionTimeout     = 10
+	embeddedEtcdPingLimit = 60
```

Comment:
```suggestion
	embeddedEtcdPingLimitDuration = 60 * time.Second
```

Reply:
should i also change the `connectionTimeout = 10 * time.Second`?